### PR TITLE
SW-1597 Update number of plants collected from to be single number

### PR DESCRIPTION
**Pull request by karthikbtf**
Terraware App 2022-09-09 11-35-04

Terraware App 2022-09-09 11-34-32

### CODE REVIEW

File: src/components/accession2/DetailPanel.tsx [hunk 1]
```diff
@@ -143,7 +143,7 @@ export default function DetailPanel(props: DetailPanelProps): JSX.Element {
               {strings.PLANT_AND_SITE}
             </Grid>
             <Grid item xs={gridRightSide}>
-              {`${strings.COLLECTED_FROM} ${accession.plantsCollectedFromMax}-${accession.plantsCollectedFromMin} ${strings.WILD_PLANT}`}
+              {`${strings.COLLECTED_FROM} ${accession.plantsCollectedFrom} ${strings.WILD_PLANT}`}
```
Comment (karthikbtf) [inline src/components/accession2/DetailPanel.tsx:146]: guessing WILD_PLANT will be changed to read from the record?

Comment (constanzauanini) [reply src/components/accession2/DetailPanel.tsx:146]: Yes, we can update it now if it's in the api

**Review by constanzauanini — APPROVED** [review]

File: src/components/accession2/DetailPanel.tsx (hunk 1 above)
Comment (karthikbtf) [reply src/components/accession2/DetailPanel.tsx:146]: yes it is the collectionSource, i will update.

File: src/components/accession2/DetailPanel.tsx [hunk 2]
```diff
@@ -143,7 +147,9 @@ export default function DetailPanel(props: DetailPanelProps): JSX.Element {
               {strings.PLANT_AND_SITE}
             </Grid>
             <Grid item xs={gridRightSide}>
-              {`${strings.COLLECTED_FROM} ${accession.plantsCollectedFrom} ${strings.WILD_PLANT}`}
+              {`${strings.COLLECTED_FROM}${numPlants === undefined ? '' : ' ' + numPlants}${
```
Comment (constanzauanini) [inline src/components/accession2/DetailPanel.tsx:150]: Thanks for doing this!

**Pull request merged**